akka-persistence-typed/src/main/mima-filters/2.8.4.backwards.excludes/EventWriter-gaps.excludes

```diff
@@ -0,0 +1,2 @@
+# internal
+ProblemFilters.exclude[Problem]("akka.persistence.typed.internal.EventWriter*")
```

Comment:
The reason is that we don't care about checking bin compat with the milestones. When 2.9.0 is out it will be added to https://github.com/akka/akka/blob/main/project/MiMa.scala#L21

akka/akka has a more advanced mima setup than many other repos, which only look for latest tag and compare with that singe version. akka/akka compares with many historical versions.

Reply:
My bad for telling Levi to do this in a previous Akka PR because I had forgotten the other projects are different. I'll PR a fix for that.